defaults/main.yml

```
@@ -31,6 +31,9 @@ codex_max_peers: 160
 codex_discovery_port: 9000
 codex_listening_port: 9000
 
+# Ethereum public key for Codex node
+codex_eth_public_key: ''
```

I'd rather comment it out and use `is defined` check.
```suggestion
#nim_codex_eth_public_key: ~
```
Also I added `nim_` prefix to variables.